README.md

```diff
@@ -1135,3 +1138,34 @@ you are using apply and auto correct.
 
 - [CMSecurityRoles_Present](Source\Examples\Resources\CMSecurityRoles\CMSecurityRoles_Present.ps1)
 - [CMSecurityRoles_Absent](Source\Examples\Resources\CMSecurityRoles\CMSecurityRoles_Absent.ps1)
+
+### CMClientPushSettings
+
+- **[String] SiteCode** _(Key)_: Specifies the Site Code for the Configuration
+  Manager site.
+- **[Boolean] EnableAutomaticClientPushInstallation** _(Write)_: Specifies whether
+  Configuration Manager automatically uses client push for discovered computers.
+- **[Boolean] EnableSystemTypeConfigurationManager** _(Write)_: Specifies whether
+  Configuration Manager pushes the client software to Configuration Manager site
+  system servers.
+- **[Boolean] EnableSystemTypeServer** _(Write)_: Specifies whether Configuration
+  Manager pushes the client software to servers.
+- **[Boolean] EnableSystemTypeWorkstation** _(Write)_: Specifies whether Configuration
+  Manager pushes the client software to workstations.
+- **[Boolean] InstallClientToDomainController** _(Write)_: Specifies whether to use
+  automatic site-wide client push installation to install the Configuration Manager
+  client software on domain controllers.
+- **[String] InstallationProperty** _(Write)_: Specifies any installation properties
```

Comment:
Add a note here so that users understand it's flat text and will take whatever they enter, even if the installation properties are not valid properties.